lib/rubocop/cop/rails/presence.rb

```diff
@@ -75,7 +75,7 @@ def on_if(node)
           unless receiver
             receiver, other = redundant_negative_receiver_and_other(node)
           end
-          return unless receiver
+          return unless receiver && other
```

Comment:
Maybe it should use blocks.
For example:

```ruby
redundant_receiver_and_other(node)  do |receiver, other|
  # ...
end

redundant_negative_receiver_and_other(node) do |receiver, other|
  # ...
end
```

Reply:
Good idea!